my-project/index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello world!</h1>
```

Comment:
Why did you create 2 index.html? You should also create .gitignore file for ignoring unnecessary files.